src/controller/python/chip/setup_payload/setup_payload.py

```diff
@@ -46,34 +45,23 @@ def AddVendorAttribute(tag, value):
 
     def ParseQrCode(self, qrCode: str):
         self.Clear()
-        err = self.chipLib.pychip_SetupPayload_ParseQrCode(qrCode.upper().encode(),
-                                                           self.attribute_visitor,
-                                                           self.vendor_attribute_visitor)
-
-        if err != 0:
-            raise ChipStackError(err)
-
-        return self
```

Comment:
The `ParseManualPairingCode` still returns `self`, so maybe this one should too.